File: EveVoid/Dto/DscanDto.cs
using System;
using System.Collections.Generic;

namespace EveVoid.Dto
{
    public class DscanDto
    {
        public int Id { get; set; }
        public int SolarSystemId { get; set; }
        public DateTime CreationDate { get; set; }

        public List<DscanShipGroupDto> DscanShipGroups { get; set; }
    }

    public class DscanShipGroupDto
    {
        public string GroupName { get; set; }
        public int GroupCount { get; set; }

        public Dictionary<string, int> ShipTypes { get; set; }
    }
}

File: EveVoid/ClientApp/src/app/api/models/dscan-dto.ts
/* tslint:disable */
import { DscanShipGroupDto } from './dscan-ship-group-dto';
export interface DscanDto {
  creationDate?: string;
  dscanShipGroups?: Array<DscanShipGroupDto>;
  id?: number;
  solarSystemId?: number;
}
